scripts/restart-tmux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+ 
+tmux send-keys -t fedimint-dev:1.4 "cargo build --bin fedimintd" C-m
+tmux send-keys -t fedimint-dev:1.4 "pkill -9 fedimintd" C-m
```

Comment:
Is that the right way around? This mixes federation and build logs, so if the build fails new epoch logs might hide that fact.

Reply:
Point taken on the ordering, I switched it around. I'm not sure if the convenience tradeoff from using -9 is worth it, so I'll defer to your judgement on that - I can certainly change it to something more graceful.

Reply:
:+1: Let's leave it till graceful shutdown is faster (fix some stuck tasks)